packages/wonder-blocks-modal/components/modal-launcher.test.js

```diff
@@ -149,4 +153,30 @@ describe("ModalLauncher", () => {
         // Now that the modal is closed, there should be no ScrollDisabler.
         expect(wrapper.find("ScrollDisabler")).toHaveLength(0);
     });
+
+    test("using `opened` and `children` should throw", () => {
+        expect(() =>
+            shallow(
+                <ModalLauncher
+                    modal={exampleModal}
+                    opened={false}
+                    onClose={() => {}}
+                >
+                    {({openModal}) => <button onClick={openModal} />}
+                </ModalLauncher>,
+            ),
+        ).toThrow("'children' and 'opened' can't be used together");
```

Comment:
I don't think this is going to work, you'll need to use the component that I created that catches React errors and hides the console.errors.

Reply:
@jeresig which tests are those being used in?

Reply:
Found it.